shunting-yard-common/pom.xml

```diff
@@ -20,6 +20,13 @@
       <artifactId>commons-lang3</artifactId>
     </dependency>
 
+    <!-- Apiary -->
+    <dependency>
+      <groupId>com.expedia.apiary</groupId>
```

Comment:
My last suggestion would be, if its possible to create a variable for this version number in shunting-yard parent `pom.xml` so that it can be consistent across the project.

Reply:
Done!